scripts/check_js_integration.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -e
+
+ROOT=`pwd`
+
+echo "Running aries-js-worker integration tests..."
+echo "----> packing aries-js-worker"
+cd $ROOT/cmd/aries-js-worker
+npm install
```

Comment:
Do we need to add NPM to the pre-requisites https://github.com/hyperledger/aries-framework-go/blob/master/docs/test/build.md#prerequisites-for-running-tests-and-demos ?

Reply:
added a couple of notes to:

* docs/test/bdd_tests.md
* docs/test/build.md